<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_ListeJeux</class>
 <widget class="QWidget" name="F_ListeJeux">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>610</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="6">
       <widget class="QComboBox" name="CBx_Auteur"/>
      </item>
      <item row="4" column="8" rowspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCommandLinkButton" name="bt_RAZ">
          <property name="text">
           <string>Remise à zéro des critères</string>
          </property>
          <property name="icon">
           <iconset resource="ImagesIcones.qrc">
            <normaloff>:/RAZ.png</normaloff>:/RAZ.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Bt_ExporterListe">
          <property name="text">
           <string>Exporter liste</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/CSV.png</normalon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="Lb_MotCle">
        <property name="maximumSize">
         <size>
          <width>900</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Mots-clés :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="Lb_Titre">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Recherche par critères</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="Bt_AjouterMotCle">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
        <property name="icon">
         <iconset resource="ImagesIcones.qrc">
          <normaloff>:/Ajouter.png</normaloff>:/Ajouter.png</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="4" colspan="2">
       <widget class="QLabel" name="Lb_Createur">
        <property name="text">
         <string>Auteurs :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QComboBox" name="CBx_MotCle"/>
      </item>
      <item row="5" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="2" colspan="2">
       <widget class="QListWidget" name="Lw_MotsCles">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="Bt_SupprimerMotCle">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ImagesIcones.qrc">
          <normaloff>:/Supprimer.png</normaloff>:/Supprimer.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="9">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="4" column="3">
         <widget class="QComboBox" name="CBx_Statut">
          <item>
           <property name="text">
            <string>Statut</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QComboBox" name="CBX_Etat">
          <item>
           <property name="text">
            <string>État</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="CBx_JoueursMax">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>Nb joueur maxi</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QComboBox" name="CBx_DateAcquisition">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Date acquisition</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QComboBox" name="CBx_PrixLoc">
          <property name="currentText">
           <string>Prix location</string>
          </property>
          <item>
           <property name="text">
            <string>Prix location</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="LE_Nom">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="CBx_JoueursMin">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>Nb joueur mini</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="CBx_Classification">
          <item>
           <property name="text">
            <string>Classification</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="CBx_Emplacement">
          <property name="currentText">
           <string>Emplacement</string>
          </property>
          <item>
           <property name="text">
            <string>Emplacement</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QComboBox" name="CBx_AgeMin">
          <item>
           <property name="text">
            <string>Âge minimum</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="Lb_Nom">
          <property name="text">
           <string>Nom :</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QComboBox" name="CBx_AgeMax">
          <item>
           <property name="text">
            <string>Âge maximum</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="5">
       <widget class="QPushButton" name="Bt_SupprimerAuteur">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ImagesIcones.qrc">
          <normaloff>:/Supprimer.png</normaloff>:/Supprimer.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="QPushButton" name="Bt_AjouterAuteur">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
        <property name="icon">
         <iconset resource="ImagesIcones.qrc">
          <normaloff>:/Ajouter.png</normaloff>:/Ajouter.png</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="6" colspan="2">
       <widget class="QListWidget" name="Lw_Auteurs">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="7">
       <widget class="QTableView" name="TbV_Recherche">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="7">
       <widget class="QLineEdit" name="LE_sql"/>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="Lb_Resultat">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="Lb_NombreResultat">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Nombre de jeux répondant à vos critères :</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ImagesIcones.qrc"/>
 </resources>
 <connections/>
</ui>
